Documentation/ceph-upgrade.md

```diff
@@ -173,37 +166,48 @@ If your `ceph status` output has deviations from the general good health describ
 be an issue that needs to be investigated further. There are other commands you may run for more
 details on the health of the system, such as `ceph osd status`.
 
-### Pod Version
-The version of a specific pod in the Rook cluster can be verified in its pod spec output. For
-example for the monitor pod `mon0`, we can verify the version it is running with the below commands:
+### Container Versions
+The container version running in a specific pod in the Rook cluster can be verified in its pod spec
+output. For example for the monitor pod `mon-b`, we can verify the container version it is running
+with the below commands:
 ```bash
-POD_NAME=$(kubectl -n $ROOK_NAMESPACE get pod -o custom-columns=name:.metadata.name --no-headers | grep rook-ceph-mon0)
+POD_NAME=$(kubectl -n $ROOK_NAMESPACE get pod -o custom-columns=name:.metadata.name --no-headers | grep rook-ceph-mon-b)
 kubectl -n $ROOK_NAMESPACE get pod ${POD_NAME} -o jsonpath='{.spec.containers[0].image}'
 ```
 
 ### All Pods Status and Version
-The status and version of all Rook pods can be collected all at once with the following commands:
+The status and container versions for all Rook pods can be collected all at once with the following
+commands:
+```sh
+kubectl -n $ROOK_SYSTEM_NAMESPACE get pod -o jsonpath='{range .items[*]}{.metadata.name}{"\n\t"}{.status.phase}{"\t"}{.spec.containers[0].image}{"\t"}{.spec.initContainers[0]}{"\n"}{end}' && \
+kubectl -n $ROOK_NAMESPACE get pod -o jsonpath='{range .items[*]}{.metadata.name}{"\n\t"}{.status.phase}{"\t"}{.spec.containers[0].image}{"\t"}{.spec.initContainers[0].image}{"\n"}{end}'
+```
+
+Rook 1.0 also introduces the `rook-version` label to Ceph controller resources. For various
```

Comment:
With #2978 are you also going to mention the `ceph-version` label?

Reply:
I think the Rook version is most important here. Even the `ceph-version` label isn't really necessary to watch for a completed Ceph upgrade.